models/form.js

```diff
@@ -4,14 +4,14 @@ var Schema = mongoose.Schema;
 var formSchema = Schema({
   _studentID: { type: Schema.Types.ObjectId, ref: 'Student' },
   period: String,
-  date: Date,
+  date: Number,
   attendance: String,
   behaviorText: String,
   warnings: String,
-  stars: String,
+  stars: Number,
   engagingContent: Number,
   engagingPeers: Number,
-  schoolBehavior: [Boolean],
+  schoolBehavior: Schema.Types.Mixed,
```

Comment:
since you made schoolBehavior a dictionary is seems like this should be embedded, like this:

```
schoolBehavior: {
    Write_up: Boolean,
    ...
}
```